apps/team-awesome-web/src/services/delete-time-entry.ts

```diff
@@ -1,7 +1,8 @@
 import * as Types from "../types";
 
 export const deleteTimeEntry = async (id: number): Promise<Types.TimeEntry> => {
-  return fetch(`http://localhost:3004/time-entries/${id}`, {
+  const baseUrl = process.env.NEXT_PUBLIC_DB_HOST;
+  return fetch(`${baseUrl}/time-entries/${id}`, {
```

Comment:
Nothing wrong with this, but in my opinion it's perfectly fine to just use the `process.env` value directly in the fetch string. Saves you having to declare the `baseUrl` const 5 times.